catalog/tests/PartitionScheme_unittest.cpp

```diff
@@ -611,6 +665,7 @@ TEST(PartitionSchemeTest, CheckRangePartitionSchemeSerialization) {
     EXPECT_EQ(blocks_in_part_scheme, blocks_in_part_scheme_from_proto);
   }
 }
+*/
```

Comment:
Let's not check in commented code. We can either get rid of these tests or try to fix it. I am not sure what these tests are trying to verify. If the functionality being tested is not really critical, we can remove the commented code, create a JIRA issue and move on.